packages/account/src/Sections/Security/SelfExclusion/self-exclusion.tsx

```diff
@@ -72,13 +72,7 @@ type TResponse = {
     };
 };
 
-const SelfExclusion = ({
-    footer_ref,
-    is_app_settings,
-    is_appstore,
-    overlay_ref,
-    setIsOverlayShown,
-}: TSelfExclusion) => {
+const SelfExclusion = ({ footer_ref, is_app_settings, overlay_ref, setIsOverlayShown }: TSelfExclusion) => {
```

Comment:
SelfExclusion is rendered from route. isnt?

can you double check if these props are valid?